content/applications/inventory_and_mrp/manufacturing/workflows/allocation.rst

```diff
@@ -0,0 +1,140 @@
+components are reserved for those that must be prioritized.
+
+In Odoo *Manufacturing*, allocation reports are used on |MOs| to assign products to specific sales
+orders |SOs|, or components to specific |MOs|. This ensure that the products or components are
+available for those orders, and are not used by mistake.
+
+Configuration
+=============
+
+To use allocation reports, the *Allocation Report for Manufacturing Orders* feature must be enabled.
+To do so, navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`, and tick
+the checkbox next to :guilabel:`Allocation Report for Manufacturing Orders`.
+
+.. image:: allocation/allocation-setting.png
+   :align: center
+   :alt: The Allocation Report for Manufacturing Orders setting.
+
+For products that are sold, it is also necessary to configure them so they can be included in |SOs|.
+To do so, begin by navigating to :menuselection:`Inventory --> Products --> Products`, and select a
+product. Under the product's name, make sure that the :guilabel:`Can Be Sold` checkbox is enabled.
+
+.. image:: allocation/can-be-sold.png
+   :align: center
+   :alt: The Can Be Sold checkbox on a product form.
+
+Allocate products
+=================
+
+To allocate products or components from an |MO| to an |SO| or different |MO|, begin by navigating to
+:menuselection:`Manufacturing --> Orders --> Manufacturing Orders`. Click :guilabel:`New` to create
+a new |MO|.
+
+On the |MO|, select a product in the :guilabel:`Product` field, and specify the quantity to be
+produced in the :guilabel:`Quantity` field. Finally, click :guilabel:`Confirm` to confirm the |MO|.
+
+The rest of the allocation workflow depends on the current on-hand quantity of the product being
+manufactured, and whether or not there are any |SOs| or |MOs| which require the product, but have
+not already been allocated units.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there are too few units
+of the product on-hand to fulfill those orders, then an :icon:`fa-list` :guilabel:`Allocation` smart
+button appears at the top of the page as soon as the |MO| is confirmed.
+
+If there **ARE** existing |SOs| and |MOs| that require the product, **AND** there **ARE** enough
+units of the product on-hand to fulfill those orders, then the :icon:`fa-list`
+:guilabel:`Allocation` smart button only appears at the top of the page once the |MO| has been
+marked as done, by clicking :guilabel:`Produce All`.
+
+.. image:: allocation/allocation-button.png
+   :align: center
+   :alt: The Allocation smart button at the top of an MO.
+
+.. note::
+   If there **ARE NOT** any existing |SOs| and |MOs| that require the product, the :icon:`fa-list`
+   :guilabel:`Allocation` smart button does not appear, even when the |MO| has been marked as done.
+
+Click the :icon:`fa-list` :guilabel:`Allocation` smart button to open the *MRP Reception Report* for
```

Comment:
```suggestion
Click the :icon:`fa-list` :guilabel:`Allocation` smart button to open the :guilabel:`MRP Reception Report` for
```
OPTIONAL
I fully get the use of italics here because, at this point, the title of the page *technically* isn't being seen. However, I am merely suggesting the use of a GUI here because, if the reader were to follow along as each step is explained, they would have, at this point, clicked the 'Allocation' smart button, which *does* reveal this title on the page. 

That said, I think both options are feasible, just thought it was worth suggesting for your consideration during your reviews.